.github/workflows/test-application.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
             matrix:
                 include:
                     - php-version: '7.2'
-                      node-version: '10'
+                      node-version: '12'
```

Comment:
It looks like the `@ckeditor/ckeditor5-core` is not compatible with `Node 10` 🙈 
https://github.com/sulu/skeleton/pull/102/checks?check_run_id=2236924856